src/geometry.rs

```diff
@@ -627,21 +629,13 @@ pub trait Geom<'a>:
     /// ```
     /// use geos::{Geom, Geometry};
     ///
-    /// let input = Geometry::new_from_wkt("MULTIPOINT((100 200), (105 202), (110 200), (140 230),
-    ///                                                (210 240), (220 190), (170 170), (170 260),
-    ///                                                (213 245), (220 190))")
+    /// let input = Geometry::new_from_wkt("MULTIPOINT(2 2, 4 2)")
     ///                   .expect("Invalid geometry");
     /// let mut expected = Geometry::new_from_wkt(
-    ///     "GEOMETRYCOLLECTION(POLYGON ((-20 50, -20 380, -3.75 380, 105 235, 105 115, 77.14285714285714 50, -20 50)),\
-    ///                          POLYGON ((247 50, 77.14285714285714 50, 105 115, 145 195, 178.33333333333334 211.66666666666666, 183.51851851851853 208.7037037037037, 247 50)),\
-    ///                          POLYGON ((-3.75 380, 20.000000000000007 380, 176.66666666666666 223.33333333333334, 178.33333333333334 211.66666666666666, 145 195, 105 235, -3.75 380)),\
-    ///                          POLYGON ((105 115, 105 235, 145 195, 105 115)),\
-    ///                          POLYGON ((20.000000000000007 380, 255 380, 176.66666666666666 223.33333333333334, 20.000000000000007 380)),\
-    ///                          POLYGON ((255 380, 340 380, 340 240, 183.51851851851853 208.7037037037037, 178.33333333333334 211.66666666666666, 176.66666666666666 223.33333333333334, 255 380)),\
-    ///                          POLYGON ((340 240, 340 50, 247 50, 183.51851851851853 208.7037037037037, 340 240)))")
-    ///                             .expect("Invalid geometry");
-    ///
-    /// let mut voronoi = input.voronoi(None::<&Geometry>, 6., false)
+    ///     "GEOMETRYCOLLECTION (POLYGON ((0 0, 0 4, 3 4, 3 0, 0 0)), POLYGON ((6 4, 6 0, 3 0, 3 4, 6 4)))")
```

Comment:
This simplified the example to use far fewer coordinates.  It looks like this:
<img width="900" alt="image" src="https://user-images.githubusercontent.com/3375604/182946528-72093393-48ad-4a50-87d5-deca12ca30f1.png">